### PR TITLE
test/add representation unittests

### PR DESCRIPTION
**Pull request by Prikshit7766**
## Checklist:

- [ ] I've added Google style docstrings to my code.
- [ ] I've used `pydantic` for typing when/where necessary.
- [ ] I have linted my code
- [ ] I have added tests to cover my changes.

### CODE REVIEW

**Review by JulesBelveze — COMMENTED** [review]
You are not testing any of the `Representation` classes right?

What I had in mind was to parametrize using all the `Representation` classes, so something like:
```python
@pytest.mark.parametrize("representation", [EthnicityRepresentation, LabelRepresentation, ...]
def test_transform(representation):
    representaton.transform(...)
```
know what I mean?

File: tests/test_representation.py [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+            "min_country_economic_representation_count": {"min_count": 10},
+            "min_country_economic_representation_proportion": {"min_proportion": 0.1},
+            "min_label_representation_count": {"min_count": 10},
+            "min_label_representation_proportion": {"min_proportion": 0.1},
+        }
+
+        transform_result = representation.transform(
+            representation, sample_data, representation_config
+        )
+
```
Comment (JulesBelveze) [inline tests/test_representation.py:109]: Hmmm the first parameter of `representation.transform` is supposed to be a `str` right? Here we passing the representation object.

**Review by JulesBelveze — CHANGES_REQUESTED** [review]
@Prikshit7766 @RakshitKhajuria I'll take it from here guys

File: tests/test_representation.py (hunk 1 above)
Comment (RakshitKhajuria) [reply tests/test_representation.py:109]: Yes. you are right

**Pull request closed without merging**